okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -344,6 +347,73 @@ static Status toGrpcStatus(ErrorCode code) {
     return ERROR_CODE_TO_STATUS.get(code);
   }
 
+  private Socket upgradeToTls(Socket socket, SSLSocketFactory sslSocketFactory) throws IOException {
+    SSLSocket sslSocket = (SSLSocket) sslSocketFactory.createSocket(
+        socket, authorityHost, address.getPort(), true /* auto close */);
+    final String[] supportedTlsProtocols = sslSocket.getSupportedProtocols();
+    Set<String> supportedTlsProtocolsSet = new HashSet<String>(supportedTlsProtocols.length);
+    for (int i = 0; i < supportedTlsProtocols.length; ++i) {
+      supportedTlsProtocolsSet.add(supportedTlsProtocols[i]);
+    }
+    List<String> enableTlsProtocols = new ArrayList<String>();
+    addIfSupported(supportedTlsProtocolsSet, enableTlsProtocols,
+        "TLSv1.2", "TLSv1.1", "TLSv1");
+    sslSocket.setEnabledProtocols(
+        enableTlsProtocols.toArray(new String[enableTlsProtocols.size()]));
+
+    final String[] supportedCiphers = sslSocket.getSupportedCipherSuites();
+    Set<String> supportedCiphersSet = new HashSet<String>(supportedCiphers.length);
+    for (int i = 0; i < supportedCiphers.length; ++i) {
+      supportedCiphersSet.add(supportedCiphers[i]);
+    }
+    List<String> enableCiphers = new ArrayList<String>();
+    addIfSupported(
+        supportedCiphersSet, enableCiphers,
```

Comment:
I think by default Netty transport is using JdkSslContext.DEFAULT_CIPHERS.

I tried to change it to use Http2SecurityUtil.CIPHERS, but it failed with:
javax.net.ssl.SSLHandshakeException: No appropriate protocol (protocol is disabled or cipher suites are inappropriate)
	at sun.security.ssl.Handshaker.activate(Handshaker.java:470)
	at sun.security.ssl.SSLSocketImpl.kickstartHandshake(SSLSocketImpl.java:1438)
	at sun.security.ssl.SSLSocketImpl.performInitialHandshake(SSLSocketImpl.java:1308)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1359)
	at sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:1343)
	at com.squareup.okhttp.OkHttpTlsUpgrader.upgrade(OkHttpTlsUpgrader.java:89)
	at io.grpc.transport.okhttp.OkHttpClientTransport.start(OkHttpClientTransport.java:209)

So I'm just put both lists here, let me know how you want me to proceed.